startup.m

```diff
@@ -0,0 +1,48 @@
+<<<<<<< Updated upstream
+% First written by : Gordon G. Parker, Michigan Technological University
+% Modifed for WEC-Sim by: Sal Husain, sal.husain@nrel.gov
+% Last modified : June 9th, 2021
+
+
+% Root directory of this running .m file
+projectRootDir = fileparts(mfilename('fullpath'));
+
+% Add project directories to path
+addpath(fullfile(projectRootDir,genpath('examples')),'-end');
+addpath(fullfile(projectRootDir,genpath('Sal')),'-end');
+addpath(fullfile(projectRootDir,genpath('source')),'-end');
+addpath(fullfile(projectRootDir,genpath('tests')),'-end');
+addpath(fullfile(projectRootDir,genpath('tutorials')),'-end');
+% addpath(fullfile(projectRootDir,genpath('models')),'-end');
+addpath(fullfile(projectRootDir,'work'),'-end');
+% addpath(fullfile(projectRootDir,genpath('scripts')),'-end');
+% % addpath(fullfile(projectRootDir,'code'),'-end');
+
+% Save Simulink-generated helper files to work
+Simulink.fileGenControl('set',...
+    'CacheFolder',fullfile(projectRootDir,'work'),...
+=======
```

Comment:
@salhus Is this line also meant to be uncommented? When I comment out it then appears that the function started on line 22 is incomplete.

Reply:
yes. sorry about that... I noticed that GitHub is adding those when I am creating a PR. I was actually trying to debug and avoid this...

Reply:
Thanks for the quick response. Does that mean line 24 is overwriting your file? If I remove line 24 it still appears that the function started on line 22 is not completed. There is still three periods after the comma on line 23.